### PR TITLE
CMR-7481, CMR-7529

### PR DESCRIPTION
**Pull request by ygliuvt**
Create platformInstrument vertices to relate platforms and instruments to collections.

### CODE REVIEW

File: graph-db/serverless/src/utils/cmr/indexPlatform.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+const gremlinStatistics = gremlin.process.statics
+
+/**
+ * create the acquiredBy edge between platform/instrument and collection
+ * @param {JSON} piVertex the platformInstrument vertex
+ * @param {Connection} gremlinConnection a connection to the gremlin server
+ * @param {Graph Node} dataset the parent collection vertex in the gremlin server
+ * @returns null
+ */
+const createAcquiredByEdge = async (piVertex, gremlinConnection, dataset) => {
```
Comment (mschmele) [inline graph-db/serverless/src/utils/cmr/indexPlatform.js:12]: Normally each of these functions would be their own file to make testing easier. I think it's worth sticking to that pattern

Comment (ygliuvt) [reply graph-db/serverless/src/utils/cmr/indexPlatform.js:12]: I think it is more readable to keep this private function in this file.

Comment (mschmele) [reply graph-db/serverless/src/utils/cmr/indexPlatform.js:12]: I disagree because it's an inconsistency in how we organize our code

Comment (macrouch) [reply graph-db/serverless/src/utils/cmr/indexPlatform.js:12]: I agree with @mschmele

Comment (ygliuvt) [reply graph-db/serverless/src/utils/cmr/indexPlatform.js]: Reluctantly refactored code. :)

**Review by mschmele — APPROVED** [review]
Thanks for making those changes

**Review by macrouch — APPROVED** [review]

**Pull request merged**